lib/svg.dart

```diff
@@ -22,7 +22,7 @@ class Svg {
   /// Renders the svg on the [canvas] using the dimensions provided by [size]
   void render(Canvas canvas, Vector2 size) {
     svgRoot.scaleCanvasToViewBox(canvas, size.toSize());
-    svgRoot.draw(canvas, null);
+    svgRoot.draw(canvas, svgRoot.viewport.viewBoxRect);
```

Comment:
was this... wrong before?

Reply:
I think it just rendered on a box to infinity, but since it is now null-safe it can't be sent null